### ISSUE TITLE
This project is dead?

### ISSUE THREAD

**Issue by nx-milos** [merlosy/ngx-material-file-input#152]
Anyone migrated to another library or is there any maintained fork?

**Comment by daemons88** [comment]
Hey!! I'm working on a new fork publishing the new package in npm to migrate at least at angular 15 when is ready I'll start publishing here all the info.

**Comment by daemons88** [comment]
I just published the package in the Angular 15 version:

- [Npm](https://www.npmjs.com/package/ngx-custom-material-file-input), (npm i ngx-custom-material-file-input)
- [Repo](https://github.com/daemons88/ngx-custom-material-file-input/tree/main)

Remember to change the references in the project to use the new ones:
ngx-custom-material-file-input, here in the [demo](https://github.com/daemons88/ngx-custom-material-file-input/blob/main/projects/demo/src/app/app.component.ts) project you have an example.

I have tested it only in the library's demo project, for now just note that the placeholder no longer works and a mat-label must be used.

If you find any problem, tell me or raise an issue in the new repo and we will review it together :D
(Just to clarify, I am not an expert in the library but given the need I created this new package to use.)

**Comment by nx-milos** [comment]
@daemons88 are you able to upgrade it to angular 17?

**Comment by daemons88** [comment]
Yes, but in like 2 weeks approx, maybe less. I need to update my project to angular 15 and then update the package to 16 if there are no problems then I can update to 17.

**Comment by daemons88** [comment]
FYI, update to Angular 17: https://www.npmjs.com/package/ngx-custom-material-file-input

**Comment by nx-milos** [comment]
@merlosy Would you be able to give maintain access to @daemons88?
@daemons88 Could you create a pull request for NG 17 upgrade?

**Comment by daemons88** [comment]
When I see the PR changes for Angular 15 being merged with the fixes commented on this [issue](https://github.com/merlosy/ngx-material-file-input/issues/126), I can consider it.
